src/index.ts

```diff
@@ -31,19 +34,23 @@ export {NgbAlert, NgbDismissibleAlert, NGB_ALERT_DIRECTIVES} from './alert/alert
 export {NgbCarousel, NgbSlide, NGB_CAROUSEL_DIRECTIVES} from './carousel/carousel';
 export {NgbCollapse, NGB_COLLAPSE_DIRECTIVES} from './collapse/collapse';
 export {NgbDropdown, NgbDropdownToggle, NGB_DROPDOWN_DIRECTIVES} from './dropdown/dropdown';
+export {NgbHighlight} from './typeahead/highlight';
```

Comment:
Do we want to export it as a public API? My understanding was that we might open it up in the future but not initially.